apps/web/components/filter/filters.tsx

```diff
@@ -0,0 +1,44 @@
+"use client"
+
+import { Container } from "../layout/container"
+import { FilterButton } from "./filter"
+import { URLParams } from "@/utils/urlBuilder"
+import { FilterSection } from "./filterSection"
+import { MinecraftEdition } from "@repo/gateway"
+import { Icon } from "../icon"
+import { useState } from "react"
+
+export const Filters = ({ urlParams }: { urlParams: URLParams }) => {
+    const [active, setActive] = useState(false)
+
+    return (
+        <Container className="flex flex-col w-full tablet:w-1/6 h-fit max-h-full shrink-0 overflow-hidden">
+            <div className="flex flex-row items-center p-4">
+                <h2>Options</h2>
+                <button className="flex tablet:hidden ml-auto" onClick={() => setActive(!active)}>
+                    <Icon iconName={active ? "close" : "fullscreen"} className="w-4 h-4 fill-mainText" />
+                </button>
+            </div>
+            <div className={`divide-y-2 border-t-2 divide-darkOverlay border-darkOverlay overflow-scroll ${active ? "flex flex-col":"hidden tablet:flex flex-col"}`}>
+                <FilterSection filter="Edition" icon="information">
+                    <FilterButton updates={{ edition: "java" as MinecraftEdition }} URLParams={urlParams}>Java</FilterButton>
+                    <FilterButton updates={{ edition: "bedrock" as MinecraftEdition }} URLParams={urlParams}>Bedrock</FilterButton>
```

Comment:
Instead of `edition: "java" as MinecraftEdition`, use `MinecraftEdition.JAVA`, etc.